integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -4023,4 +4026,79 @@ public static void createIngressAndRetryIfFail(int maxRetries,
       }
     }
   }
+
+  /**
+   * Build image with unique name, create corresponding docker secret and push to registry.
+   *
+   * @param dockerFileDir directory where dockerfile is located
+   * @param baseImageName base image name
+   * @param namespace image namespace
+   * @param secretName docker secretname for image
+   * @param extraDockerArgs user specified extra docker args
+   * @return image name
+   */
+  public static String createPushImage(String dockerFileDir, String baseImageName,
```

Comment:
The method name could be more like createImageAndPushToRepo

Reply:
done